.travis.yml

```diff
@@ -56,12 +56,6 @@ env:
     - >-
       BUILD_TYPE=centaurHoricromtalEngineUpgradePapiV2alpha1
       BUILD_MARIADB=10.3
-    - >-
-      BUILD_TYPE=centaurPapiUpgradePapiV1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV1
-      BUILD_MYSQL=5.7
```

Comment:
I think this means we can also delete lines 272-277 from `test.inc.sh` (https://github.com/broadinstitute/cromwell/blob/develop/src/ci/bin/test.inc.sh#L272) as well as `papi_v1_v2alpha1_upgrade_application.conf` (https://github.com/broadinstitute/cromwell/blob/develop/src/ci/resources/papi_v1_v2alpha1_upgrade_application.conf).

Reply:
Or maybe only some of that, if we're keeping `testCentaurPapiUpgradeNewWorkflowsPapiV1.sh`?